### DISCUSSION TITLE
Bit Rates and Sound Quality

### DISCUSSION THREAD

**Discussion by GreenGryphon** [audiamus/AaxAudioConverter#126] [Q&A] [answered]
Yes, I understand that upscaling doesn't improve sound quality.  That said, when I convert a book to MP3 using your program, it defaults to 32kpbs (at 22 kHz).  Yet, when I convert the SAME BOOK to MP3 using another program with 64kpbs (at 44kHz), the narrator's voice sounds fuller and richer.  I would rather have the better sound quality even at the expense of larger file size.  (I prefer MP3 for portability.)

Your program is superior in every other way, and I would like to use it, but this is a sticking point.  So, what am I missing?  If the AAX file has better sound to begin with, I need the MP3 bit rate high enough to capture it.  Why is it defaulting to 32kpbs?  Is there an algorithm that sets it?  If so, how?   Is there any way I can adjust it?  Thanks in advance for any help you can give!  Great program!

**Comment by audiamus** [comment, marked as answer]
The default for conversion to MP3 from AAX (which is MP4/AAC) are the sample and bit rates found for the AAC audio stream. If you do not activate any bit rate settings in AAX Audio Converter, those defaults will be applied. That's the way it's handled by FFmpeg which does all the audio processing.

Nonetheless, 32 kb/s seems rather low to me. When you open the AAX file in AAX Audio Converter it will show you the AAC sample and bit rates. It should be 32 kb/s there, too. How did you download the AAX file? You could try [Book Lib Connect](https://github.com/audiamus/BookLibConnect) at the highest setting "extreme" to find out what Audible offers as max quality.

**Reply by GreenGryphon** [reply]
I initially downloaded the file into Audible Manager, but noticed it had stopped working correctly.  Instead of appearing in my Audible Manager, the downloaded file appeared in my general Downloads folder, without being "decoded" into intelligible speech.  Even after "importing" it into Audible Manager, clicking on it produced only garbled static.  So I called Audible Customer Support and they informed me that Audible Manager is no longer supported.  That's when I started looking around for another program and found yours.

Yes, I thought about trying Book Lib Connect but, when I try to run it, it tells me I need to install .NET, and the link takes me to NET 5.0.  Well, I am elderly and no longer very tech-savvy, so I am still on Windows 7, and it doesn't look like it supports NET 5.0.  I know I don't want to risk screwing up my OS, so maybe I am just out of luck.  But I would appreciate any insights you might have.  Many thanks!

**Reply by audiamus** [reply]
Windows 7 does support .Net 5. You just need to download the correct runtime package: .NET **Desktop** Runtime 5.0.15 for Windows x64.
This appears to be the direct link: 
https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/runtime-desktop-5.0.15-windows-x64-installer

**Comment by GreenGryphon** [comment]
I installed Net 5 from the link you provided and it worked flawlessly on my old Windows 7 machine, and Book Lib Connect started working. So I used the "Extreme" setting to download the SAME book again and my MP3 player now reported 64kb/s (at 22 kHz).  The next book I downloaded came in at 128kb/s (at 44 kHz), with notably improved sound quality on both.  So, thank you SO much!!!

One little oddity was that the little preview pane in AAX Audio Converter said I could expect 63kb/s and 127kb/s, respectively, one less kb/s than the standard MP3 increments.  Yet my MP3 player reported 64 and 128 during playback.  *shrug*

Anyway, kudos for a great product!  I love all the extras, like capturing the chapter titles, etc.  This has saved my association with Audible!

**Comment by LeeBinder** [comment]
native Audible aax = aac m4b = encrypted m4a/mp4 @ 128 kb/s @ 44.1 kHz sample rate
